app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
Inherit the `Dwarf` class from `ABC` since it does not implement all abstract methods of the `Player` thus being an abstract class as well.